made intakeCone also work for cubes

AlenaNguyen opened the pull request:
included an if statement to make intake speed change depending on the game piece

AlenaNguyen committed:
added if statement to intakeCone to include cubes

intake speed/direction changes depending on game piece

AlenaNguyen committed:
Merge branch 'main' into create-intake-cube-command

TaylerUva commented on src/main/java/frc/robot/RobotPreferences.java:
```diff
@@ -137,7 +138,8 @@ public static final class prefElevator {
     public static final SN_DoublePreference elevatorMinPos = new SN_DoublePreference("elevatorMinPos", 0);
 
-    public static final SN_DoublePreference elevatorIntakingPos = new SN_DoublePreference("elevatorIntakingPos", 2);
+    public static final SN_DoublePreference elevatorIntakingPos = new SN_DoublePreference("elevatorConeIntakingPos",
```
The variable name and key string do not match

AlenaNguyen committed:
fixed elevatorIntakingPos key string

AlenaNguyen committed:
Merge branch 'create-intake-cube-command' of https://github.com/FRCTeam3255/2023_Offbot_Code into create-intake-cube-command

alicekuznetsov commented on src/main/java/frc/robot/RobotContainer.java:
```diff
@@ -109,8 +110,11 @@ private void configureBindings() {
+    conOperator.btn_RightBumper.onTrue(new IntakeGamePiece(subWrist, subIntake, subElevator, GamePiece.CONE));
+
+    // Intake Cube (LB)
+    conOperator.btn_RightBumper.onTrue(new IntakeGamePiece(subWrist, subIntake, subElevator, GamePiece.CUBE));
```
// Intake Cube (LB)
you typed btn_RightBumper

alicekuznetsov commented on src/main/java/frc/robot/RobotPreferences.java:
```diff
@@ -137,7 +138,8 @@ public static final class prefElevator {
     public static final SN_DoublePreference elevatorMinPos = new SN_DoublePreference("elevatorMinPos", 0);
 
-    public static final SN_DoublePreference elevatorIntakingPos = new SN_DoublePreference("elevatorIntakingPos", 2);
+    public static final SN_DoublePreference elevatorIntakingPos = new SN_DoublePreference("elevatorIntakingPos",
```
Could you add a unit to this? If there is no unit you don't need to

alicekuznetsov commented on src/main/java/frc/robot/commands/IntakeGamePiece.java:
```diff
@@ -6,30 +6,41 @@
+      intakeSpeed = prefIntake.intakeConeSpeed.getValue();
+    } else if (gamepiece == GamePiece.CUBE) {
+      intakeSpeed = prefIntake.intakeCubeSpeed.getValue();
+    }
```
Could you add a fallback condition? I know it should never be anything else but redundancy doesn't hurt

alicekuznetsov committed:
Addressed my own comments

alicekuznetsov committed:
Merge branch 'main' of https://github.com/FRCTeam3255/2023_Offbot_Code into create-intake-cube-command

alicekuznetsov committed:
Finish addressing Eli's comments

- Removing limit switch
- Wrist should always be stowed unless its safe to out-take

alicekuznetsov committed:
Fix a comment

alicekuznetsov committed:
Set current game piece collected at end of intaking

alicekuznetsov committed:
Update Intake.java

alicekuznetsov committed:
tested

alicekuznetsov committed:
Clean up timee
